CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## Version 1.14.2
+* Release date: May 20, 2022
+* Release status: GA
+
+### What's new in 1.14.2
+* Adds ability to create new SQL Docker instances when publishing a SQL database project
```

Comment:
why is this here? this feature is added to SQL project extension not mssql.

Reply:
Ah, you're right.  I was thinking about the purpose of the release, not the actual contents of vscode-mssql.